_posts/2016-01-11-33C3.md

```diff
@@ -0,0 +1,52 @@
+---
+layout:     post
+title:      33C3 - Chaos Computer Congress 2016
+author:     jake_roberts
+date:       2016-01-11 08:31:00
```

Comment:
Is this an old post that is just being published? If so, does the date want updating to current so it appears as new content or leaving as is?